specification/service-map/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/arm-service-map.json

```diff
@@ -1673,6 +1721,10 @@
             "hypervisor": {
               "$ref": "#/definitions/HypervisorConfiguration",
               "description": "Hypervisor-related configuration. Present only when 'virtualizationState' is `hypervisor`."
+            },
+            "hosting": {
+              "$ref": "#/definitions/HostingConfiguration",
+              "description": "Hosting-related configuration. Present if hosting information is discovered for the VM."
```

Comment:
From the comment "Present if hosting information is discovered for the VM" it seems this property is readonly, if so add `readOnly` attribute